src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -145,7 +145,7 @@ public abstract class GATKTool extends CommandLineProgram {
     /**
      * Our source of reference data (null if no reference was provided)
      */
-    ReferenceDataSource reference;
+    protected ReferenceDataSource reference;
```

Comment:
Ideally we would keep this package-protected. It's meant to be encapsulated for walker traversals (descendants of `WalkerBase`), and exposed for direct descendants of `GATKTool` (ie., non-walkers). Non-walker tools can access this field via the inherited `directlyAccessEngineReferenceDataSource()` method.  @mcovarr / @lbergelson , would it be a significant hardship for the GVS branch to migrate to using that method?

Reply:
Would it be okay if we do the following in `ExtractTool.java`? Only the three subclasses of `ExtractTool.java` reference `reference`.

```
protected ReferenceDataSource reference = directlyAccessEngineReferenceDataSource();
```

Reply:
@mcovarr Of course, that's totally fine! If those are the only usages in the GVS branch, then, would you be ok with @lbergelson changing this field back to package-accessible?

Reply:
Sure that would be fine, thanks!